src/pages/home/report/ReportActionItemCreated.js

```diff
@@ -52,12 +52,11 @@ const ReportActionItemCreated = (props) => {
             onClose={() => Report.navigateToConciergeChatAndDeleteReport(props.report.reportID)}
         >
             <View style={StyleUtils.getReportWelcomeContainerStyle(props.isSmallScreenWidth)}>
-                <View pointerEvents="none" style={StyleUtils.getReportWelcomeBackgroundImageViewStyle(props.isSmallScreenWidth)}>
-                    <ImageBackground
-                        source={EmptyStateBackgroundImage}
-                        style={StyleUtils.getReportWelcomeBackgroundImageStyle(props.isSmallScreenWidth)}
-                    />
-                </View>
+                <Image
+                    source={EmptyStateBackgroundImage}
+                    style={StyleUtils.getReportWelcomeBackgroundImageStyle(props.isSmallScreenWidth)}
+                />
+                <View pointerEvents="none" style={StyleUtils.getReportWelcomeBackgroundImageViewStyle(props.isSmallScreenWidth)} />
                 <View
                     accessibilityLabel="Chat welcome message"
                     style={styles.p5}
```

Comment:
```suggestion
                <View
                    accessibilityLabel="Chat welcome message"
                    style={[styles.p5,StyleUtils.getReportWelcomeBackgroundImageViewStyle(props.isSmallScreenWidth)]}
```

We can clean this up a bit with @aimane-chnaif's suggestion to make this have a marginTop rather than an extra View here with a fixed height.

However, for this to work you'll need to change `getReportWelcomeBackgroundImageViewStyle` to return `marginTop` instead of `height`. I also would suggest renaming the function since it's not quite right now. Maybe something like `getReportWelcomeTopMarginStyle` or something like that?

Other reviewers, let me know if this sounds good to you! I didn't test on every platform but checked Safari and Chrome on desktop, as well as iOS and it seems like it has the same behavior, with cleaner code.